fiberplane-templates/src/convert/mod.rs

```diff
@@ -166,7 +166,7 @@ pub fn cells_to_snippet(cells: &[Cell]) -> String {
 
 fn write_preamble(writer: &mut CodeWriter) {
     writer.println(
-        "// For documentation on Fiberplane Templates, see: https://docs.fiberplane.com/templates",
+        "// For documentation on Fiberplane Templates, see: https://docs.fiberplane.com/docs/templates",
```

Comment:
Though it's redirecting properly, we can omit the docs subdomain
```suggestion
        "// For documentation on Fiberplane Templates, see: https://fiberplane.com/docs/templates",
```